### DISCUSSION TITLE
【灌水】非常好绳网，使我打开加速器

### DISCUSSION THREAD

**Discussion by hsuehyaung** [share121/inter-knot#1166] [其他]
话说有人见过法厄同吗，好久都找不到人了
![image](https://github.com/user-attachments/assets/046f4104-4434-4470-8958-b04f8f68043d)

**Comment by Jack1Nero1** [comment]
不知道，有传言说最近治安局抓绳匠的力度变大了，不会真被正义逮捕了吧？

**Comment by lingyushen** [comment]
<h1 style="font-size: 4rem; color: #2ecc71">实名上网，传奇绳匠无所畏惧</h1>

**Comment by chaochao6667** [comment]
法厄同这么厉害 不应该会被这么轻易的抓住吧？说不定练小号开新篇了呢？

**Comment by fairy-B** [comment]
### 我是法厄同的狗！！！###

**Comment by convant** [comment]
上次治安官不是说抓住了吗？楼主没看新闻？

**Comment by suiry-D** [comment]
楼主有电脑用的免费vpn吗？求！
